api/machine/machine.proto

```diff
@@ -82,12 +82,12 @@ message ApplyConfigurationRequest {
   bytes data = 1;
   // replaced by mode
   bool on_reboot = 2 [
-    (common.remove_deprecated_field) = "v1.1",
```

Comment:
bumped the deprecation, as at the very least we need to have it for the upgrade tests, and I think we should drop these pretty important things couple of cycles later